addon/services/flexberry-logging.js

```diff
@@ -39,30 +81,34 @@ export default Ember.Service.extend({
    */
   init() {
     this._super(...arguments);
-    this.set('flexberryStore', getOwner(this).lookup('service:store'));
-    this.set('serverLogEnabled', true);
+    this.set('_serverLogEnabled', true);
+    for (let enumName in this.logLevelEnums) {
+      let level = this.logLevelEnums[enumName];
+      this.enumsLoglevel[level] = enumName;
+    }
   },
 
   /**
-    Logger message to console and store.
+    Logger message to console and store
 
     @method flexberryLogger
-    @param {String} levelName Category name  - ERROR, WARN, LOG, INFO, DEBUG, DEPRECATION.
-    @param {String} message Message content.
-    @param {String} formattedMessage Full message content in JSON format.
-   */
-  flexberryLogger(levelName, message, formattedMessage) {
+    @param priority - priority
```

Comment:
You have to add parameter type:
"@param {String} msg A description of..." (http://yui.github.io/yuidoc/syntax/).
Description starts from uppercase letter and ends with a dot.